diff -ruN approx.orig/util.ml approx/util.ml
--- approx.orig/util.ml	2016-07-02 15:51:24.547192965 +0200
+++ approx/util.ml	2016-07-02 15:52:52.861698192 +0200
@@ -170,9 +170,9 @@
 (* Decompression programs for supported compression formats *)
 
 let decompressors =
-  [".bz2", "/bin/bzcat";
-   ".gz", "/bin/zcat";
-   ".xz", "/usr/bin/xzcat"]
+  [".bz2", "bzcat";
+   ".gz", "zcat";
+   ".xz", "xzcat"]
 
 let compressed_extensions = List.map fst decompressors

